codemeta.json

```diff
@@ -122,10 +128,9 @@
   "maintainer": [
     {
       "@type": "Person",
-      "givenName": "Adriaan M.",
-      "familyName": "Dokter",
-      "email": "amd427@cornell.edu",
-      "@id": "https://orcid.org/0000-0001-6573-066X"
+      "givenName": "Alexander",
```

Comment:
we should find a way to have both of us be able to trigger maintenance tasks with CRAN. We can set up an email alias at Cornell that forwards to both of us, will add as a to do issue.

Reply:
agreed